portals/application/bin/template.php

```diff
@@ -12,12 +12,11 @@
 // Define path to application directory
 $route = '';
 if ( APPLICATION_ENV != 'localdev'){
```

Comment:
I think this environment no longer exists. We only use development and production.